pkgs/games/dwarf-fortress/wrapper/dfhack.in

```diff
@@ -2,8 +2,8 @@
 
 source @dfInit@
 
-for i in dfhack.init-example dfhack-config/default hack/* stonesense/*; do
-  update_path "$i"
+for i in *.init *.init-example dfhack-config/default dfhack-config/init hack/* stonesense/*; do
```

Comment:
The location of the init scripts was changed from `*.init` to `dfhack-config/init/*` in 0.47.05-r7, so we support both.